### PR TITLE
Introduce new krew plugin apparmor-manager

### PR DESCRIPTION
**Pull request by Kaizhe**
```
$ kubectl krew install --manifest apparmor-manager.yaml
Installing plugin: apparmor-manager
Installed plugin: apparmor-manager
WARNING: You installed a plugin from the krew-index plugin repository.
   These plugins are not audited for security by the Krew maintainers.
   Run them at your own risk.
```

```
$ kubectl apparmor-manager
kubectl apparmor-manager manages AppArmor service and profiles enforcement on worker nodes through syncing with AppArmorProfile CRD in Kubernetes cluster

Usage:
  apparmor-manager [command]

Available Commands:
  enabled     Check AppArmor status on worker nodes
  enforced    Check AppArmor profile enforcement status on worker nodes
  help        Help about any command
  init        Install CRD in the cluster and AppArmor services on worker nodes
  sync        Synchronize the AppArmor profiles from the Kubernetes database (etcd) to worker nodes

Flags:
  -h, --help           help for apparmor-manager
      --level string   Log level (default "info")

Use "apparmor-manager [command] --help" for more information about a command.
```

### CODE REVIEW

**Comment by ahmetb** [comment]
We actually prefer apparmor-manager if that’s ok with you. A name like “aa” isn’t easily understandable and therefore doesn’t follow naming guidelines.

**Comment by Kaizhe** [comment]
> We actually prefer apparmor-manager if that’s ok with you. A name like “aa” isn’t easily understandable and therefore doesn’t follow naming guidelines.

Done. Thanks!

**Comment by ahmetb** [comment]
Would you mind fixing your usage string to say “kubectl xxx” when it’s executed as a plugin? Not absolutely necessary but would be preferred. You can look at os.Args[0]

**Comment by Kaizhe** [comment]
> Would you mind fixing your usage string to say “kubectl xxx” when it’s executed as a plugin? Not absolutely necessary but would be preferred. You can look at os.Args[0]

Done, thanks!

File: plugins/apparmor-manager.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  name: apparmor-manager
+spec:
+  version: v0.0.1
+  homepage: https://github.com/sysdiglabs/kube-apparmor-manager
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/sysdiglabs/kube-apparmor-manager/releases/download/v0.0.1/kube-apparmor-manager_v0.0.1_darwin_amd64.tar.gz
```
[the remark this answers is not in the document]
Comment (Kaizhe) [reply plugins/apparmor-manager.yaml]: yes, it does

File: plugins/apparmor-manager.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  - selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+    uri: https://github.com/sysdiglabs/kube-apparmor-manager/releases/download/v0.0.1/kube-apparmor-manager_v0.0.1_linux_amd64.tar.gz 
+    sha256: 0ba7ce50749878792d7643b7695b8f05bfbabf072b3ffc45fc31052bd22cc43d 
+    bin: kube-apparmor-manager
+  shortDescription: Manage AppArmor profiles for cluster.
+  description: |
+    Manage AppArmor profiles through AppArmorProfile CRD and sync across all worker nodes.
```
[the remark this answers is not in the document]
Comment (Kaizhe) [reply plugins/apparmor-manager.yaml]: Done.

File: plugins/apparmor-manager.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    sha256: 3a28dddd7ccbae22d7cd40942231620596600fb80003de9c076aec8b01928457 
+    bin: kube-apparmor-manager
+  - selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+    uri: https://github.com/sysdiglabs/kube-apparmor-manager/releases/download/v0.0.1/kube-apparmor-manager_v0.0.1_linux_amd64.tar.gz 
+    sha256: 0ba7ce50749878792d7643b7695b8f05bfbabf072b3ffc45fc31052bd22cc43d 
+    bin: kube-apparmor-manager
+  shortDescription: Manage AppArmor profiles for cluster.
```
[the remark this answers is not in the document]
Comment (Kaizhe) [reply plugins/apparmor-manager.yaml]: Done.

**Comment by Kaizhe** [comment]
@ahmetb let me know if you have any other comments. Thanks!

**Pull request merged**